src/solver/variable/categories.h

```diff
@@ -213,15 +207,6 @@ inline void FileLevelToStreamShort(StreamT& out, int fileLevel)
     case de:
         out += "de";
         break;
-    case is:
```

Comment:
Should not **DataLevel::standard** disappear as well ?

Reply:
I don't see uses of Category::standard or DataLevel::standard in the code, maybe we should remove it

Reply:
you can't see it because you removed : 
```C++
    if (CDataLevel & Category::standard)
        RunStandard(list, results, numSpace);
```

Reply:
**Category::standard** and  **DataLevel::standard** are the same, it seems

Reply:
Besides, removing this **enum** value (**standard**) will force us to change the values of other **enum** values of **DataLevel::standard** : 
**DataLevel::area** now should be **1**, **DataLevel::thermalAggregate** should be **2**, and so on